pages/index.js

```diff
@@ -1,7 +1,21 @@
+import useSWR from "swr";
+
+import ArtPieces from "@/components/ArtPieces";
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+
 export default function HomePage() {
+  const { data } = useSWR("https://example-apis.vercel.app/api/art", fetcher);
+
+  if (!data) {
+    return null;
+  }
```

Comment:
you can replace this with checks for whether there was an error or it is still loading:

if (error) return <div>failed to load</div>
if (isLoading) return <div>loading...</div>

(you would need to destructure those two variables from useSWR in addition to data)

If there is no error and it is not loading anymore, the data will be there and you don't need that check anymore